x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/detection_page_filters.cy.ts

```diff
@@ -108,7 +107,7 @@ const assertFilterControlsWithFilterObject = (
 };
 
 // FLAKY: https://github.com/elastic/kibana/issues/171890
-describe.skip(`Detections : Page Filters`, { tags: ['@ess', '@serverless'] }, () => {
+describe(`Detections : Page Filters`, { tags: ['@ess', '@serverless'] }, () => {
   beforeEach(() => {
     deleteAlertsAndRules();
     createRule(getNewRule({ rule_id: 'custom_rule_filters' }));
```

Comment:
just wondering, is there a value in having the `{rule_id: 'custom_rule_filters' }` here?